src/reducers/__tests__/helpers/index.js

```diff
@@ -0,0 +1,7 @@
+import reduce from 'lodash-es/reduce';
+
+import reducer from '../../ui';
```

Comment:
Since this is intended as a helper for all reducer tests, I don’t think we want to specifically import and use the `ui` one :slightly_smiling_face: